client/src/components/ProtectedRoute.js

```diff
@@ -1,17 +1,23 @@
 import React from "react";
-import { Redirect, Route } from "react-router-dom";
+import { Route, Redirect } from "react-router-dom";
+import PropTypes from "prop-types";
 
 import AuthService from "../services/auth.service";
 
-const ProtectedRoute = ({ component: Component, ...rest }) => {
+const ProtectedRoute = (props) => {
```

Comment:
Please let the original changes with `...rest` to be able to pass all props we may want to the route :-)